[2025-07-30T13:49:30.304+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-30T13:49:30.505+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_yyyyyyyyyyyyyyyyy.extract_data_task scheduled__2025-06-25T00:00:00+00:00 [queued]>
[2025-07-30T13:49:30.512+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_yyyyyyyyyyyyyyyyy.extract_data_task scheduled__2025-06-25T00:00:00+00:00 [queued]>
[2025-07-30T13:49:30.513+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-30T13:49:30.529+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_data_task> on 2025-06-25 00:00:00+00:00
[2025-07-30T13:49:30.543+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=6539) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-30T13:49:30.545+0000] {standard_task_runner.py:63} INFO - Started process 6542 to run task
[2025-07-30T13:49:30.543+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_yyyyyyyyyyyyyyyyy', 'extract_data_task', 'scheduled__2025-06-25T00:00:00+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/dag_generator_python.py', '--cfg-path', '/tmp/tmpsevmpt5_']
[2025-07-30T13:49:30.546+0000] {standard_task_runner.py:91} INFO - Job 69: Subtask extract_data_task
[2025-07-30T13:49:30.594+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_yyyyyyyyyyyyyyyyy.extract_data_task scheduled__2025-06-25T00:00:00+00:00 [running]> on host 14614e29d246
[2025-07-30T13:49:30.683+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_yyyyyyyyyyyyyyyyy' AIRFLOW_CTX_TASK_ID='extract_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-06-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-25T00:00:00+00:00'
[2025-07-30T13:49:30.684+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-30T13:49:30.694+0000] {__init__.py:73} INFO - 執行 extract_data (資料載入層)
[2025-07-30T13:49:30.695+0000] {__init__.py:77} INFO - 呼叫演算法 'test_csv' 來自模組 'processor.extract.readCSV'
[2025-07-30T13:49:30.696+0000] {logging_mixin.py:188} INFO - 3f2a4b93-b237-4545-bd50-5004d863567c.csv
[2025-07-30T13:49:30.715+0000] {__init__.py:92} INFO - 資料已存入 Redis，key=***_data:dag_yyyyyyyyyyyyyyyyy.extract_data_task
[2025-07-30T13:49:30.731+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-30T13:49:30.732+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-30T13:49:30.740+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_yyyyyyyyyyyyyyyyy, task_id=extract_data_task, run_id=scheduled__2025-06-25T00:00:00+00:00, execution_date=20250625T000000, start_date=20250730T134930, end_date=20250730T134930
[2025-07-30T13:49:30.800+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-30T13:49:30.823+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-30T13:49:30.825+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
